internal/git/util_test.go

```diff
@@ -453,7 +453,7 @@ AAAECl1AymWUHNdRiOu2r2dg97arF3S32bE5zcPTqynwyw50HAtto0bVGTAUATJhiDTjKa
 	repo, err := cloneRepo(context.Background(), gitSync, cloneOptions, metric)
 	assert.NoError(t, err)
 	assert.NotNil(t, repo)
-	err = FileExists(repo, "data.yaml") // data.yaml default file exists in docker git
+	err = FileExists(repo, "readme.md") // readme.md default file exists in docker git
```

Comment:
This is working because we always have `readme.md` by default?

Reply:
yes ,as we are not putting any yaml files . For bdd tests they are being put by git pod which does got operations . So for testing purpose have added this file